src/components/loader/loader.stories.mdx

```diff
@@ -42,6 +43,18 @@ This example of the Loader component demonstrates how it will appear as default.
   </Story>
 </Canvas>
 
+### isActive
```

Comment:
Is this story needed? It looks like isActive defaults to false, so the presence of the isActive prop isn't really important.

I guess you've added this to enable a Chromatic snapshot, but won't that be covered by the new "inside button" examples below? You could always add the disable example to those if it's important. If we do keep this story then I'd rather have a title and comments that explain what it's supposed to be showing because I'm still not really understand it! (Might be just me...)

Reply:
I'll get it deleted. I originally added it for Chromatic snapshot reasons but as you say this will be superseded by the new `inside button` examples 😄